### DISCUSSION TITLE
feat: include webui userId in completions request

### DISCUSSION THREAD

**Discussion by sam-trost** [open-webui/open-webui#14297] [Ideas]
# Feature Request

OpenAI API supports applications including an anonymized `user` to support moderation / abuse.  See more info in the api spec for chat completions.

> A unique identifier representing your end-user, which can help OpenAI to monitor and detect abuse. [Learn more](https://platform.openai.com/docs/guides/safety-best-practices/end-user-ids).

**Comment by tjbck** [comment]
PR Welcome! https://github.com/open-webui/open-webui/pull/6366#issuecomment-2432881229
